service/coupon/src/main/resources/application.yml

```
@@ -0,0 +1,26 @@
+          show_sql: true # sql 로깅
+          format_sql: true # SQL 문 정렬하여 출력
+          highlight_sql: true # SQL 문 색 부여
+  #        use_sql_comments: true # 콘솔에 표시되는 쿼리문 위에 어떤 실행을 하려는지 HINT 표시
+
+server:
+  port: 19097
+
+eureka:
+  client:
+    service-url:
+      defaultZone: ${EUREKA_SERVER_URL}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)